### PR TITLE
Feature card comments

### PR DESCRIPTION
**Pull request by BigDinDonMan**

### CODE REVIEW

File: app/controllers/card_comment_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    end
+
+    private
+
+    def get_target_card
+        current_user.boards.find(params[:board_id]).lists.find(params[:list_id]).cards.find(params[:card_id])
+    end
+
+    def comment_params
+        params.permit(:content, :card_id)
```
Comment (3elk4) [inline app/controllers/card_comment_controller.rb:41]: We need also to addict comment to user

File: app/controllers/card_comment_controller.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        get_target_card.card_comments.find(params[:id]).update(comment_params)
+
+        render json: {success: "Comment updated successfully"}, status: 200
+    end
+
+    def delete
+        ensure_params_exist
+        get_target_card.card_comments.find(params[:id]).delete
+
+        render json: {success: "comment deleted successfully, sarge!"}
```
Comment (3elk4) [inline app/controllers/card_comment_controller.rb:31]: why we need sarge here? 🤔

File: config/routes.rb [hunk 3]
```diff
@@ -36,6 +36,13 @@
   post 'archive_card', to: 'card#archive'
   post 'restore_card', to: 'card#restore'
 
+  #card comments
+  post 'create_comment', to: 'card_comment#create'
+  post 'update_comment', to: 'card_comment#update'
+  post 'delete_comment', to: 'card_comment#delete'
+  get 'get_card_comment/:board_id/:list_id/:card_id/:id', to: 'card_comment#get'
+  get 'get_card_comments/:board_id/:list_id/:card_id/:id', to: 'card_comment#get_all'
```
Comment (3elk4) [inline config/routes.rb:44]: i think we don't need the last id while getting all comments

**Review by 3elk4 — CHANGES_REQUESTED** [review]
Overall, good job! 🎉

File: app/controllers/card_comment_controller.rb (hunk 2 above)
Comment (BigDinDonMan) [reply app/controllers/card_comment_controller.rb:31]: yes we do, its necessary :)

**Review by 3elk4 — APPROVED** [review]
🔥

**Pull request merged**